src/descriptor_runner/webdnn.ts

```diff
@@ -191,6 +192,18 @@ export async function load(directory: string, initOption: InitOption = {}): Prom
     backendOrder = backendOrder.slice();
     if (backendOrder.indexOf('fallback') === -1) backendOrder.concat(['fallback']);
 
+    // FIXME: User-defined transformDelegate is overridden
```

Comment:
Sorry, my review comment was bad. Could you register this delegate function as default function instead of [this no-op function](https://github.com/mil-tokyo/webdnn/blob/master/src/descriptor_runner/fetch.ts#L11)?